pacman/model/partitioner_splitters/splitter_one_to_one_legacy.py

```diff
@@ -45,23 +45,17 @@ class SplitterOneToOneLegacy(AbstractSplitterCommon):
         "_sdram"]
 
     def __init__(self):
-        super().__init__(type(self).__name__)
+        super().__init__()
         self._machine_vertex = None
         self._vertex_slice = None
         self._sdram = None
 
-    def __str__(self):
-        return f"SplitterOneToOneLegacy for {self._governed_app_vertex}"
-
-    def __repr__(self):
-        return self.__str__()
-
     @overrides(AbstractSplitterCommon.set_governed_app_vertex)
     def set_governed_app_vertex(self, app_vertex):
         super().set_governed_app_vertex(app_vertex)
         self._vertex_slice = Slice(0, self._governed_app_vertex.n_atoms - 1)
         self._sdram = (
-            self._governed_app_vertex.get_sdram_used_by_atoms(
+             self._governed_app_vertex.get_sdram_used_by_atoms(
```

Comment:
github is saying an extra space has been added here which seems to mess up the formatting...

Reply:
reversed